Common/Securities/Option/OptionStrategies.cs

```diff
@@ -1151,6 +1151,129 @@ DateTime expiration
+            Symbol canonicalOption,
+            decimal higherStrike,
+            decimal lowerStrike,
+            DateTime expiration
+            )
+        {
+            CheckCanonicalOptionSymbol(canonicalOption, "PutBackspread");
+            CheckExpirationDate(expiration, "PutBackspread", nameof(expiration));
+
+            if (higherStrike <= lowerStrike)
+            {
+                throw new ArgumentException($"PutBackspread: strike prices must be in descending order, {nameof(higherStrike)}, {nameof(lowerStrike)}");
+            }
+
+            return new OptionStrategy
+            {
+                Name = "Put Backspread",
+                Underlying = canonicalOption.Underlying,
+                CanonicalOption = canonicalOption,
+                OptionLegs = new List<OptionStrategy.OptionLegData>
+                {
+                    new OptionStrategy.OptionLegData
+                    {
+                        Right = OptionRight.Put, Strike = higherStrike, Quantity = -1, Expiration = expiration
+                    },
+                    new OptionStrategy.OptionLegData
+                    {
+                        Right = OptionRight.Put, Strike = lowerStrike, Quantity = 2, Expiration = expiration
+                    }
+                }
+            };
+        }
+
+        /// <summary>
+        /// Method creates new Short Call Backspread strategy, that consists of two calls with the same expiration but different strikes.
+        /// It involves buying the lower strike call, while shorting twice the number of the higher strike call.
+        /// </summary>
+        /// <param name="canonicalOption">Option symbol</param>
+        /// <param name="lowerStrike">The strike price of the long call</param>
+        /// <param name="higherStrike">The strike price of the short call</param>
+        /// <param name="expiration">Option expiration date</param>
+        public static OptionStrategy ShortCallBackspread(
+            Symbol canonicalOption,
+            decimal lowerStrike,
+            decimal higherStrike,
+            DateTime expiration
+            )
+        {
+            return InvertStrategy(CallBackspread(canonicalOption, lowerStrike, higherStrike, expiration), "Short Call Backspread");
+        }
+
+        /// <summary>
+        /// Method creates new Short Put Backspread strategy, that consists of two puts with the same expiration but different strikes.
+        /// It involves buying the higher strike put, while selling twice the number of the lower strike put.
+        /// </summary>
+        /// <param name="canonicalOption">Option symbol</param>
+        /// <param name="higherStrike">The strike price of the long put</param>
+        /// <param name="lowerStrike">The strike price of the short put</param>
+        /// <param name="expiration">Option expiration date</param>
+        /// <returns>Option strategy specification</returns>
+        public static OptionStrategy ShortPutBackspread(
```

Comment:
Nice ty. Can the regression algorithms actually use these option strategies, else if feels like they are not being tested against? The unit tests are okay, but let's assert they work when used like the users will use them 👍

Reply:
sure I changed to use option strategies helper class to order